### ISSUE TITLE
Helium Brent failure

### ISSUE THREAD

**Issue by thorade** [CoolProp/CoolProp#887]
These are extreme inputs, but Brent should not fail, I believe.
If at all, there should be a different error message.

```Python
import CoolProp as CP
from CoolProp.CoolProp import PropsSI
# Check: CoolProp version
print(CP.__version__)
print(CP.__gitrevision__)

FluidName = 'Helium'
# Constants, triple and critical data
R = PropsSI('GAS_CONSTANT',FluidName)
MM = PropsSI('MOLAR_MASS',FluidName)
Rs = R/MM
T_crt = PropsSI('T_CRITICAL',FluidName)
T_trp = PropsSI('T_TRIPLE',FluidName)
p_max = PropsSI('P_MAX',FluidName)
d_max = PropsSI('D','T',T_trp,'P',p_max,FluidName)
```
Probably, p_max is higher than the melting pressure for that temperature?

**Comment by ibell** [comment]
Helium is an extreme fluid in general, but the consistency plots look more
or less ok:
http://www.coolprop.dreamhosters.com/binaries/sphinx/fluid_properties/fluids/Helium.html#consistency-plots

**Comment by thorade** [comment]
This can be closed as duplicate of #743 and #744
